src/api.rs

```diff
@@ -1245,6 +1332,8 @@ impl<T: Pixel> ContextInner<T> {
     }
     for i in 0..(self.output_frameno - 1) {
       self.frame_invariants.remove(&i);
+      self.gop_output_frameno_start.remove(&i);
```

Comment:
if you have more fields indexed by the same value maybe it is time to consider a struct.

Reply:
I think the eventual plan was to get rid of at least one of those two items once `build_frame_properties` is further simplified? @tdaede what do you think?

Reply:
Yeah I'd like to remove at least input/output_frameno_start but am not 100% sure I can remove both. They could be combined into a struct.

Currently frame_invariants is inserted at a different time than gop_*start, so it can't be put in a struct. Maybe once we drop the redundant frame_invariant writes then it will be ok.